official/AnalisiDeiRequisiti/res/sections/utenteA.tex

```diff
@@ -231,11 +234,12 @@ \subsubsection{Visualizzazione messaggio di errore: password errata}
         \hline
         
         \textbf{Attori Primari} & Utente autenticato \\ 
-        \textbf{Scopo e Descrizione} & L'Utente Autenticato visualizza un messaggio di errore durante \glossaryItem{procedura} di modifica del profilo dovuto all'inserimento di una password errata. \\ 
+        \textbf{Scopo e Descrizione} & L'Utente Autenticato visualizza un messaggio di errore durante la \glossaryItem{procedura} di modifica del profilo dovuto all'inserimento di una password errata. \\ 
         
         \textbf{Precondizioni}  & L'Utente Autenticato ha inserito una password errata nella \glossaryItem{procedura} di cambio della password. \\ 
         
         \textbf{Postcondizioni} & L'Utente Autenticato ha visualizzato il messaggio di errore. \\ 
+        \textbf{Scenario principale} & 1. L'Utente Autenticato visualizza un messaggio di errore durante la \glossaryItem{procedura} di modifica del profilo dovuto all'inserimento di una password errata.  \\
```

Comment:
- [ ] @EmanueleC questo scenario principale è uguale a quello in linea 237, sei sicuro che sia corretto?